src/thorin/be/c/c.cpp

```diff
@@ -481,8 +481,9 @@ void CCodeGen::emit_epilogue(Continuation* cont) {
     } else if (auto callee = cont->callee()->isa_continuation(); callee && callee->is_basicblock()) { // ordinary jump
         assert(callee->num_params() == cont->num_args());
         for (size_t i = 0, size = callee->num_params(); i != size; ++i) {
-            if (auto arg = emit_unsafe(cont->arg(i)); !arg.empty())
```

Comment:
But that should already take care of "invalid" args, no? We return the empty string for unit and mem objects.

Reply:
we were also emitting variables for mem

Reply:
but yes, should work, but it doesn't. I'll look into it. you are right.